.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Marktext: Main",
+      "program": "${workspaceFolder}/.electron-vue/dev-runner.js"
```

Comment:
I did not tested this on Windows, but do we not have to set the windows runtime executable?

```
      "program": "${workspaceFolder}/.electron-vue/dev-runner.js",
      "windows": {
            "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron.cmd"
      }
```

Reply:
I don't think we need windows `runtimeExecutable` config, because we don't run electron directly, we run `electro-vue` that wrap this.